resources/views/backend/books/edit.blade.php

```diff
@@ -0,0 +1,86 @@
+@extends('backend.layouts.master')
+@section('title')
+    {{ __('Dashboard') }}
+@endsection
+@section('content')
+<div class="content-wrapper">
+  <section class="content-header">
+    <h1>
+      {{ __('Edit Book') }}
+    </h1>
+    <ol class="breadcrumb">
+      <li>
+        <a href="#">
+          <i class="fa fa-dashboard"></i> {{ __('Home') }}</a>
+      </li>
+      <li class="active">{{ __('Books') }}</li>
+    </ol>
+  </section>
+  <section class="content">
+      <div class="row">
+        <div class="col-md-12">
+          <div class="box box-primary">
+            <form role="form" method="" enctype="multipart/form-data">
+              {{ csrf_field() }}
+              <div class="box-body">
+                <div class="form-group">
+                  <label>{{ __('List Category') }}</label>
+                  <div class="row">
+                    <div class="col-xs-2">
+                      <select class="form-control" name="category" >
+                        <option value="">option 1</option>
+                        <option value="">option 2</option>
+                        <option value="">option 3</option>
+                        <option value="">option 4</option>
+                        <option value="">option 5</option>
+                      </select>
+                    </div>
+                  </div>
+                </div>
+                <div>
+                  <div class="row">
+                    <div class="col-xs-6">
+                      <div class="form-group">
+                        <label for="exampleInputTitle">{{ __('Title') }}</label>
+                        <input type="text" class="form-control" name="title" placeholder="{{ __('Title') }}">
```

Comment:
Chú ý `for="inputTitle"` trong label nha, `inputTitle` là ID của input mà label đó đại diện. trong input title nên có 1 cái là `id="inputTitle"` nữa.
Khi mình click vào label (chữ title) => nó sẽ focus vào input để mình nhận data.
==> Mục đích là tránh thừa code, thừa kí tự => trong code php á, dư 1 dấu cách cũng ko được chấp nhận đó nha. :D

Sửa mấy cái ở dưới nữa hỷ! ;)